src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -989,11 +984,10 @@ void ConnectionHandlerImpl::PairFromKey(
     uint8_t* session_id) const {
   *connection_handle = key & 0xFF00FFFF;
   *session_id = key >> 16;
-  SDL_LOG_DEBUG(
-
-      "ConnectionHandle:" << static_cast<int32_t>(*connection_handle)
-                          << " Session:" << static_cast<int32_t>(*session_id)
-                          << " for key:" << static_cast<int32_t>(key));
+  SDL_LOG_DEBUG("ConnectionHandle:"
+                << static_cast<int32_t>(*connection_handle)
+                << " Session:" << static_cast<int32_t>(*session_id)
+                << " for key:" << static_cast<int32_t>(key));
```

Comment:
```suggestion
  SDL_LOG_DEBUG("ConnectionHandle: "
                << static_cast<int32_t>(*connection_handle)
                << " Session: " << static_cast<int32_t>(*session_id)
                << " for key: " << static_cast<int32_t>(key));
```
Might be good to have a space between the key and value